Interpolation/examples/Interpolation/nn_coordinates_2.cpp

```diff
@@ -8,7 +8,6 @@
 #include <utility>
 #include <vector>
 
-typedef CGAL::Exact_predicates_inexact_constructions_kernel K;
```

Comment:
That must be an error.

Reply:
Woah, thanks! At least it broke something...